Antidote/ActiveCallViewController.m

```diff
@@ -283,6 +283,28 @@ - (void)installConstraints
     }];
 }
 
+#pragma mark - Constraints movement
+
+- (void)moveVideoButtonToRight
+{
+    [self.videoButton remakeConstraints:^(MASConstraintMaker *make) {
+        make.top.equalTo(self.controlsContainerView);
+        make.centerX.equalTo(self.speakerButton);
+        make.size.equalTo(kButtonSide);
+        make.bottom.equalTo(self.speakerButton.top).with.offset(-k3ButtonGap);
+    }];
+}
+
+- (void)moveVideoButtonToCenter
+{
+    [self.videoButton remakeConstraints:^(MASConstraintMaker *make) {
+        make.top.equalTo(self.controlsContainerView);
+        make.centerX.equalTo(self.controlsContainerView);
+        make.size.equalTo(kButtonSide);
+        make.bottom.equalTo(self.microphoneButton.top).with.offset(-k3ButtonGap);
```

Comment:
Instead of this you can `activate` or `deactivate` specific constraints. I suggest you to 
- create "resume" button in `viewDidLoad` as other buttons
- install constraints for all buttons in `installConstraints` method
- make "resume" button hidden
- deactivate "video" `right` constraint

When there would be need to show "resume" button, you can deactivate "video" `center` constrait and activate `right` one.

```
@property (strong, nonatomic) MASConstraint *videoCenterXConstraint;
@property (strong, nonatomic) MASConstraint *videoRightConstraint;

...

- (void)viewDidLoad
{
    ...

    [self hideResumeButton];
}

...
- (void)installConstraints
{
    ...

    [self.resumeButton makeConstraints:^(MASConstraintMaker *make) {
        make.right.equalTo(self.videoButton.left).with.offset(-k3ButtonGap);
        make.centerY.equalTo(self.videoButton);
        make.size.equalTo(kButtonSide);
    }];

    [self.videoButton makeConstraints:^(MASConstraintMaker *make) {
        make.top.equalTo(self.controlsContainerView);
        make.size.equalTo(kButtonSide);
        make.bottom.equalTo(self.microphoneButton.top).with.offset(-k3ButtonGap);

        self.videoCenterXConstraint = make.centerX.equalTo(self.controlsContainerView);
        self.videoRightConstraint = make.right.equalTo(self.controlsContainerView);·
    }];

    ...
}

...

- (void)showResumeButton
{
    self.resumeButton.hidden = NO;
    [self.videoCenterXConstraint deactivate];
    [self.videoRightConstraint activate];
}

- (void)hideResumeButton
{
    self.resumeButton.hidden = YES;
    [self.videoCenterXConstraint activate];
    [self.videoRightConstraint deactivate];
}
```